packages/contracts-bedrock/src/L1/SuperchainConfig.sol

```diff
@@ -0,0 +1,238 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+import { Initializable } from "@openzeppelin/contracts/proxy/utils/Initializable.sol";
+import { ISemver } from "src/universal/ISemver.sol";
+import { Types as T } from "src/libraries/Types.sol";
```

Comment:
With the import causing bindings issues, in the future we can use the `abigen`  flag for excluding types, its `abigen -exc` but that would require a change to the bindings build script/config. If you would like to take it on to get the preferable API, that is cool but also its all good if not

Reply:
I think we need to aggressively cut scope here, resolving :)